### DISCUSSION TITLE
Please, update the Yarn Version

### DISCUSSION THREAD

**Discussion by RSickenberg** [community/community#13238] [Code Security]
As a developer and core leader on some projects, I usually rely on dependabot once per week to remind me to update the deps (if I didn't do it before).

Thus, dependabot uses Yarn V1 so it shifts my `yarn.lock` file and had to implement a workflow (inspired by this: https://gist.github.com/amacneil/60bf679f357bad9d62103cfdc86cbd74) and the issue https://github.com/dependabot/dependabot-core/issues/1297 is still not fixed.

Thanks for it!

**Comment by exvuma** [comment]
The nature of yarn v2 makes this issue pretty tricky for the team to implement. Curious if supporting yarn v3 would help ?  Or if moving to yarn 3 would be out of the question? @RSickenberg

**Reply by RSickenberg** [reply]
Hello @exvuma, thank you for your answer! 

`Yarn V3` would fit perfectly too. In fact, it would be better.

**Reply by christian-hawk** [reply]
Any plans?
